src/components/Comments.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router";
+import { fetchComments } from "../api";
+import LoadingComments from "./LoadingComments";
+
+const Comments = () => {
+    const {review_id} = useParams();
+    const [comments, setComments] = useState([]);
+    const [loadingComments, setLoadingComments] = useState(true);
+
+    useEffect(() => {
+        console.log(review_id)
+        fetchComments(review_id)
+        .then((comment) => {
+            setComments(comment)
+            setLoadingComments(false);
+        })
+    }, [review_id])
+    
+    if(loadingComments){return <LoadingComments/>};
+
+    return <section>
+        <ul className='comment-grid'>
```

Comment:
When there are no comments for a review, you could let the user know about it with another conditional rendering. At least they know that the lack of comments is not a problem with the website.